src/logger/index.ts

```diff
@@ -58,28 +56,34 @@ export default class ConsoleLogger implements ILogger {
   };
 
   public succeedIcon = (message: string): void => {
-    this.info(`${logSymbols.success} ${message}`);
+    this.info(`✅ ${message}`);
```

Comment:
One option to removing dependency on log-symbols which, as in the case of ora, requires dynamic imports.  I'm fine with using emoji, unless there's a preference or reason to do otherwise.